### ISSUE TITLE
New Community Curator information to add to curator page

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#1843]
Here is a list of our community curators as part of our 2022 Intake who have created an account with us. In the next week or two, the final members of the intake will have their accounts and then I will submit another ticket to add them.

Please could you add this information to the page listing [our community curators](https://fairsharing.org/community_curation/our_curators)? Thanks!

However, please do NOT update their ORCID profiles yet, as I need to confirm their domain name with them first, and check that they all have orcid tokens on our system.

Name | orcid | user id | domain | organisation name | organisation abbreviation | organisation id | social media
-- | -- | -- | -- | -- | -- | -- | --
Antonella Delmestri | 0000-0003-0388-3403 | 7747 | Medical Informatics | University of Oxford | | 3112 | LinkedIn: antonella-delmestri
Daniel Pereira Costa | 0000-0001-7271-430X | 7754 | Catalysis | Centro de Química Estrutural |CQE| 3699 | LinkedIn: daniel-costa-64a54a1a3/
Deborah Thorpe | 0000-0002-2307-8770 | 7663 | Humanities | University College Cork|UCC |3677|
Muhammad Rafiq | 0000-0002-8291-2569 | 7730 | Library and Information Science | Institute of Information Management, University of the Punjab|Institute of Information Management, PU| 3692|
Enrique Wulff | 0000-0001-8104-6147 |7667 | Earth Sciences  | Institute of Marine Sciences |ICM| 3708|
Gabriel Pelletier | 0000-0003-3257-0545 | 7670 | Neuroscience | Tanenbaum Open Science Institute, The Montreal Neurological Institute and Hospital and McGill University |TOSI, The Neuro, McGill | 2840, 1896, 1802| Twitter: neuropelletier
Geneviève MICHAUD | 0000-0001-9288-2888 | 2439 | Humanities and Social Science | Sciences Po & the National Center for Scientific Research |Sciences Po, CNRS| 2521, 1974 | twitter: @GenevievMichaud 
Guillaume Gay | 0000-0002-5444-5246 |  7729 |  Microscopy | University of Montpellier | |3101 | Twitter: MorphoLg
Joao Moreira | 0000-0002-4547-7000 | 7678 | Computer Science | University of Twente | |3143 |
Lars Vilhuber | 0000-0001-5733-8932 | 5216 | Economics | Cornell University | |614
Maja Dolinar | 0000-0002-8307-6486 | 7669 | Humanities and Social Science | Social Sciences Research Institute | SSRI|3710
Mitrea Geta | 0000-0002-2411-1798 | 7660 | Humanities and Social Science | University `Stefan cel Mare` of Suceava | USV| 3676 | https://twitter.com/GetaMitreaRo  https://www.linkedin.com/in/geta-mitrea/ 
Sara Espinoza | 0000-0002-5902-650X | 7751 | Material sciences and catalysis | Society for Chemical Engineering and Biotechnology |DECHEMA| 3711
Shukadev Amgain | 0000-0002-5113-6878 | 7745 | Social Sciences and Humanities | Open Knowledge Nepal || 3698
Stephen Serjeant | 0000-0002-0517-7943 | 7735 | Physics | The Open University |OU| 2816

Antonella Delmestri:
![adelmestri](https://user-images.githubusercontent.com/143586/192258982-e6c0fdb5-3e47-4a2d-9e79-61285b1aa9c4.jpg)

Daniel Pereira Costa:
![WhatsApp Image 2022-09-12 at 14 59 53](https://user-images.githubusercontent.com/143586/192259070-982633fb-c900-49fa-a3f4-185560416384.jpeg)

Genevieve Michaud:
![Michaud](https://user-images.githubusercontent.com/143586/192259191-e57d3e09-5757-4980-b600-9563d0cf42b6.jpg)

Muhammad Rafiq's photo:
![IMG_20220219_173533352](https://user-images.githubusercontent.com/143586/192235664-c2b20045-9efb-403d-9715-a587a42c5a55.jpg)

Guillaume Gay:
![pp](https://user-images.githubusercontent.com/143586/192251159-c26fa0af-fd4c-4d92-a1dc-d381c32bae59.jpg)

Geta Mitrea
![geta](https://user-images.githubusercontent.com/143586/192256827-44840fae-37f3-4ca0-97aa-e09549017e1d.jpeg)

And also, please add these photos for our early adopters:

Malin:
![PXL_20220806_100014122 PORTRAIT](https://user-images.githubusercontent.com/143586/192261182-1195ff57-06f9-4a77-b18b-22ad697a22d6.jpg)

Lindsey:
![image001](https://user-images.githubusercontent.com/143586/192261258-4798f25d-030d-4509-aa30-019697898ad7.jpg)

**Comment by prakhyatox** [comment]
Few queries

1) Are these entries on the top of the existing entries in list?
2) Are all the new entries are ```Early Adopters```?
3) Are we expecting the profile pics for all the entries or this is only we have so far?

**Comment by allysonlister** [comment]
Thanks @prakhyatox 

1. The only overlap you have between this ticket and the already existing entries are: Guillaume Gay, who is already on the webpage, so I doubt there is any new info in this ticket, but you could double check; and 2 photos of our early adopters (Malin and Lindsey) that we didn't have before.
2. All of the new entries are NOT early adopters, but rather our first "normal" intake for 2022 and therefore do not need the early adopter tag
3. We might get a few more profile pics once they see the new page, but we might not, so some of them will just have the default icon you've used so far.

Thanks!

**Comment by prakhyatox** [comment]
> 
> 1. The only overlap you have between this ticket and the already existing entries are: **_Guillaume Gay_**, who is already on the webpage, so I doubt there is any new info in this ticket, but you could double check; and 2 photos of our early adopters (Malin and Lindsey) that we didn't have before. ----

Re: Actually its Gabriel Pelletier, and there is new info as he along with Geneviève MICHAUD are associated with more than one organisation for which I have update the logic to display if there are more than one organisation for a one curator

**Comment by allysonlister** [comment]
Thank you for correcting me, and apologies - I think I may be trying to write these things too quickly - thanks!

**Comment by prakhyatox** [comment]
1) 
![image](https://user-images.githubusercontent.com/109139583/192291781-fef3ab92-ca33-4588-9cc7-a4a5fbbbacb1.png)
3 Organisations , 4 ids.  Please confirm


2) As per data few organisations were not found. Below are their ids
3692, 3708, 3710, 3676, 7751

**Comment by allysonlister** [comment]
Hmm that's interesting!

Gabriel: For now I've removed 3682 as it is a professional affiliation rather than a place he works at, thank you for pointing out the issue here.

Organisations not found:

https://fairsharing.org/organisations/3692 exists as does https://fairsharing.org/organisations/3708, https://fairsharing.org/organisations/3710, https://fairsharing.org/organisations/3676 . Could it be that you don't have the most up to date version of the db content?

organisation 7751 was a typo on my part, sorry (it was the user id). That row should have an organisation id of 3711 - I've updated it in the original comment above as well.

Thanks!
